dev/docs/UseBaseAPIDocs.md

```diff
@@ -0,0 +1,110 @@
+#### The useBaseAPI Interface
+
+```ts
+/**
+ * UseBaseAPIOptions extends Trees/ReqOptions
+ * these options are used only in the instantiation
+ * of a new UseBaseAPI composable
+ */
+export interface UseBaseAPIOptions extends ReqOptions {
```

Comment:
I wonder if 1. we could write our implementation such that it's importable here rather than redefining and 2. if it's even possible to import code here from elsewhere in the project. Just food for thought, nothing to do now